packages/ts/form/src/Binder.ts

```diff
@@ -35,21 +35,21 @@ const _validationRequestSymbol = Symbol('validationRequest');
  * @param <M> is the type of the model that describes the structure of the value
  */
 export class Binder<T, M extends AbstractModel<T>> extends BinderNode<T, M> {
-  private [_defaultValue]: T;
+  private [_defaultValue]!: T; // Initialized in the `read()` method
 
-  private [_value]: T;
+  private [_value]: T = this[_emptyValue];
```

Comment:
It also requires suppressing
```suggestion
  private [_value]: T!;
```